### DISCUSSION TITLE
v23.0.3

### DISCUSSION THREAD

**Discussion by thaJeztah** [moby/moby#45271] [Announcements]
## 23.0.3

> **Note**
>
> Due to an issue with CentOS 9 Stream's package repositories, packages for
> CentOS 9 are currently unavailable. Packages for CentOS 9 may be added later,
> or as part of the next (23.0.4) patch release.

### Bug fixes and enhancements

- Fixed a number of issues that can cause Swarm encrypted overlay networks
  to fail to uphold their guarantees, addressing [CVE-2023-28841](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28841),
  [CVE-2023-28840](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28840), and
  [CVE-2023-28842](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28842).
    - A lack of kernel support for encrypted overlay networks now reports
      as an error.
    - Encrypted overlay networks are eagerly set up, rather than waiting for
      multiple nodes to attach.
    - Encrypted overlay networks are now usable on Red Hat Enterprise Linux 9
      through the use of the `xt_bpf` kernel module.
    - Users of Swarm overlay networks should review [GHSA-vwm3-crmr-xfxw](https://github.com/moby/moby/security/advisories/GHSA-vwm3-crmr-xfxw)
      to ensure that unintentional exposure has not occurred.

### Packaging Updates

- Update containerd to [v1.6.20](https://github.com/containerd/containerd/releases/tag/v1.6.20).
- Update runc to [v1.1.5](https://github.com/opencontainers/runc/releases/tag/v1.1.5).

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v23.0.3'>v23.0.3</a>.</em>

**Comment by zdz2019** [comment]
Packaging Updates invalid.

**Reply by thaJeztah** [reply]
Could you elaborate? (what is invalid?)

**Reply by zdz2019** [reply]
Source code (v23.0.3.tar.gz)
The version numbers of related files were not updated.
